glide.yaml

```diff
@@ -24,3 +24,4 @@ import:
   subpackages:
   - stack
 - package: github.com/go-errors/errors
+- package: gopkg.in/guregu/null.v3
```

Comment:
As we're managing version of third-party libraries by glide, it's not appropriate to use `gopkg.in`. Otherwise It'll be double-managed.
So let it be `github.com/guregu/null`, and specify `version: 3.1`.